src/components/LazyLog/README.md

```diff
@@ -35,3 +35,28 @@ consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?
   <LazyLog extraLines={1} enableSearch text={text} caseInsensitive />
 </div>
 ```
+
+Log viewing using a websocket
+
+```jsx harmony
+const url = 'wss://echo.websocket.org';
+let socket = null;
+
+<div style={{ height: 200, width: 902 }}>
+  <button style={{margin: 5}} onClick={() => socket && socket.send(JSON.stringify({ message: '[taskcluster 2018-11-14 21:08:32.452Z] Worker Group: us-east-1' }))}>ping</button>
+  <LazyLog
+    enableSearch
+    url={url}
+    websocket
+    websocketOptions={{
+      onOpen: (e, sock) => {
+          socket = sock; sock.send(JSON.stringify({message: "Socket has been opened!"}))
+        },
+      formatMessage: (e) => {
+          return `${JSON.parse(e).message}\n`;
```

Comment:
Having users write `\n` is a bit inconvenient. Is there a way to handle this internally? Perhaps

```jsx
socket.addEventListener('message', e => {
  let msg = formatMessage ? formatMessage(e.data) : e.data;

  if (!msg.endsWith('\n')) {
    msg += '\n';
  }

  emitter.emit('data', msg);
});
```

Reply:
good suggestion! I added something similar to this in my latest commit.

Thank you for all of your patience and thorough reviews!